scripts/install.sh

```diff
@@ -5,68 +5,68 @@ OS="$(uname -s)"
 THEMIS_CONTRACT_REPO=${THEMIS_CONTRACT_REPO:-'git@github.com:informalsystems/themis-contract.git'}
 
 unixlike_font_installed() {
-  SPEC="$1"
```

Comment:
This is just from my autoformatter. Let me know if you'd rather I revert it.

Reply:
Not a problem 👍